pkg/controllers/work/patch_util.go

```diff
@@ -114,14 +114,17 @@ func setModifiedConfigurationAnnotation(obj runtime.Object) error {
 func getOriginalConfiguration(obj runtime.Object) ([]byte, error) {
```

Comment:
Hi Ryan! Just a nit here: this function seems to be no longer returning an error in any branch.

Reply:
good point, fixed